src/wiki-table-editor.jsx

```diff
@@ -5,7 +5,24 @@ import * as Table from 'reactabular-table';
 import * as dnd from 'reactabular-dnd';
 import cloneDeep from 'lodash/cloneDeep';
 import findIndex from 'lodash/findIndex';
+import draggableRow from './draggable-row.jsx';
 
+const DndRow = draggableRow('tr');
+class DndCell extends React.Component {
+  render() {
+    let props = cloneDeep(this.props);
+    let canDragCell = props.canDragCell;
+    delete props.connectDragSource;
+    delete props.canDragCell;
```

Comment:
I'm not sure what you mean. This component doesn't have any state.

Though, these `delete`s could be cleaned up a bit. It's just making a copy of `props` without the `canDragCell` or `connectDragSource` properties.